test/fixtures/tvos_app/bwx_spec.json

```diff
@@ -11,6 +11,7 @@
         "VALIDATE_WORKSPACE": false
     },
     "configuration": "darwin_x86_64-fastbuild-ST-d53d69b6b8c1",
+    "custom_xcode_schemes": null,
```

Comment:
Do we have an example of the DTO coming through the spec?

Reply:
My bad. I thought that generator had fixture tests. That is the one spot that we have custom schemes defined. I can add it to one of the examples. Are you OK with me adding custom schemes to `multiplatform`? It seems like we can exercise the target-platform logic, if we add some there.

Reply:
The generator does have fixture tests.

Reply:
I'm also fine with adding some to `multiplatform`.

Reply:
The generator fixtures now contain the schemes using the updated `xcodeproj_fixture` macro.